apps/docs/content/guides/auth/auth-mfa/webauthn.mdx

```diff
@@ -0,0 +1,195 @@
+---
+id: 'auth-mfa-webauthn'
+title: 'Multi-Factor Authentication (WebAuthn)'
+description: 'Add an additional layer of security with passwordless authentication using biometrics and security keys.'
+---
+
+## What is WebAuthn?
+
+WebAuthn (Web Authentication) is a web standard that enables secure, passwordless authentication using public-key cryptography. It allows users to log in to websites and applications using built-in authenticators like fingerprint sensors, facial recognition, or external devices like security keys.
+
+### When to Use WebAuthn?
+
+Consider implementing WebAuthn when:
+
+1. You want to enhance security beyond traditional passwords
+2. You aim to provide a more user-friendly authentication experience
+3. You need to reduce costs associated with SMS-based authentication
+4. You want to offer a hardware backup to existing authentication factors
+
+WebAuthn is particularly suitable for security-conscious applications that require a user-friendly, passwordless solution.
+
+### Pricing and Availability
+
+WebAuthn MFA is part of the Auth Advanced MFA Add-on. Here's the breakdown of pricing:
+
+- Cost: $75 per month for the first project in the organization
+- Additional projects: $10 per month each
+- Included in the Add-on: Phone and other advanced MFA features
+
+## Types of Authenticators
+
+An authenticator is a component that securely generates and stores cryptographic keys associated with a particular relying \***\*party (web page origin) and a credential \*\***ID (a unique identifier for the key). The authenticator uses these keys to authenticate the user to the relying party without directly storing user-specific data like the user ID.
```

Comment:
formatting looks off in the preview 
<img width="973" alt="image" src="https://github.com/user-attachments/assets/e9a9614b-7e21-40e9-b798-2f26cce7d727">